### ISSUE TITLE
Laravel 7 HTTP Client

### ISSUE THREAD

**Issue by DominikGeimer** [LKDevelopment/hetzner-cloud-php-sdk#51]
Hello,

I have a Laravel 7 an and I use the new HTTP Client from Laravel 7 (https://laravel.com/docs/7.x/http-client). 

When I try to install your package via composer I get an error because of the already existing Guzzle:

```
Problem 1
    - Can only install one of: guzzlehttp/guzzle[6.5.x-dev, 7.0.1].
    - Can only install one of: guzzlehttp/guzzle[7.0.1, 6.5.x-dev].
    - Can only install one of: guzzlehttp/guzzle[6.5.x-dev, 7.0.1].
    - lkdevelopment/hetzner-cloud-php-sdk v2.2.0 requires guzzlehttp/guzzle ^6.3 -> satisfiable by guzzlehttp/guzzle[6.5.x-dev].
    - Installation request for lkdevelopment/hetzner-cloud-php-sdk ^2.2 -> satisfiable by lkdevelopment/hetzner-cloud-php-sdk[v2.2.0].
    - Installation request for guzzlehttp/guzzle (locked at 7.0.1, required as ^7.0) -> satisfiable by guzzlehttp/guzzle[7.0.1].
```

Is there any way to make this work?

**Comment by LKaemmerling** [comment]
Hey @DominikGeimer,

thank you! This will be fixed with the next release (i will just fix a few other things, than i will publish a new version).

**Comment by LKaemmerling** [comment]
@DominikGeimer i just released v2.2.1 which includes a fix for this.
